custom/enikshay/nikshay_datamigration/management/commands/create_enikshay_cases.py

```diff
@@ -0,0 +1,210 @@
+from django.core.management import BaseCommand
+from django.db import transaction
+
+from casexml.apps.case.const import CASE_INDEX_EXTENSION
+from casexml.apps.case.mock import CaseFactory, CaseStructure, CaseIndex
+from corehq.apps.locations.models import SQLLocation
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+
+from custom.enikshay.nikshay_datamigration.models import PatientDetail, Outcome, Followup
+from dimagi.utils.decorators.memoized import memoized
+
+
+class EnikshayCaseFactory(object):
+
+    domain = None
+    patient_detail = None
+
+    def __init__(self, domain, patient_detail):
+        self.domain = domain
+        self.patient_detail = patient_detail
+        self.factory = CaseFactory(domain=domain)
+        self.case_accessor = CaseAccessors(domain)
+
+    @transaction.atomic
```

Comment:
Does this work as expected with couch cases? The enikshay domain is currently still using the couch backend.